src/basic/xfork.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include <sched.h>
+
+/* ======================================================================= */
```

Comment:
this comment should be dropped btw, there's nothing really here that it could separate from...